src/assets/scss/style.scss

```diff
@@ -1,11 +1,11 @@
 // Vendor
-@import 'vendor/_normalize.scss';
```

Comment:
If you decide to change the quotes, make it consistent in the whole file, instead of only a few lines